.ci/after_success.sh

```diff
@@ -0,0 +1,171 @@
+  echo ">>> DONE($?): git config user.email <<<"
+
+  echo ">>> DEBUG >>> git config --list > $(git config --list)"
+
+  # Prevent log warning by explicitly setting push strategy
+  echo ">>> START: git config --global push.default simple <<<"
+  git config --global push.default simple
+  echo ">>> DONE($?): git config --global push.default simple <<<"
+
+  # Npm auth
+  echo ">>> START: npm config set username ${NPM_CONFIG_USERNAME}"
+  npm config set username "${NPM_CONFIG_USERNAME}"
+  echo ">>> DONE($?): npm config set username ${NPM_CONFIG_USERNAME}"
+
+  echo ">>> START: npm config set username ${NPM_CONFIG_EMAIL}"
+  npm config set email "${NPM_CONFIG_EMAIL}"
+  echo ">>> DONE($?): npm config set username ${NPM_CONFIG_EMAIL}"
+
+  echo ">>> START: npm config set //registry.npmjs.org/:_authToken=... <<<"
+  npm config set "//registry.npmjs.org/:_authToken=${NPM_TOKEN}"
+  echo ">>> DONE($?): npm config set //registry.npmjs.org/:_authToken=... <<<"
+
+  echo ">>> DEBUG >>> npm whoami > $(npm whoami)"
+  echo ">>> DEBUG >>> npm config list > $(npm config list)"
+
+  # Fetch tags
+  echo ">>> START: git fetch --tags <<<"
+  git fetch --tags
+  echo ">>> DONE($?): git fetch --tags <<<"
+
+  # Set upstream branch
+  echo ">>> START: git branch -u origin/$TRAVIS_BRANCH <<<"
+  git branch -u origin/$TRAVIS_BRANCH
+  echo ">>> DONE($?): git branch -u origin/$TRAVIS_BRANCH <<<"
+
+  echo ">>>>>>>>>>>>>>>>>>>>>>>>>>> DONE: SETTING GIT/NPM FOR RELEASE <<<<<<<<<<<<<<<<<<<<<<<<<<<"
+
+#
+# Build app
+#
+
+  echo ">>>>>>>>>>>>>>>>>>>>>>>>>>> START: BUILDING APP <<<<<<<<<<<<<<<<<<<<<<<<<<<"
+
+  # Run build task
+  echo ">>> START: yarn build <<<"
+  yarn build
+  echo ">>> DONE($?): yarn build <<<"
+
+  echo ">>>>>>>>>>>>>>>>>>>>>>>>>>> DONE: BUILDING APP <<<<<<<<<<<<<<<<<<<<<<<<<<<"
+
+#
+# Release
+#
+
+echo ">>>>>>>>>>>>>>>>>>>>>>>>>>> START: RELEASE <<<<<<<<<<<<<<<<<<<<<<<<<<<"
+
+# !!! this doesn't work when runing using yarn command see https://github.com/yarnpkg/yarn/issues/2935 !!!
+
+echo ">>> START: lerna-semantic-release pre > analyze commits, set versions and create tags"
+./node_modules/.bin/lerna-semantic-release pre
+echo ">>> DONE($?): lerna-semantic-release pre > analyze commits, set versions and create tags"
+
+echo ">>> START: lerna-semantic-release post > generate CHANGELOG"
+./node_modules/.bin/lerna-semantic-release post
+echo ">>> DONE($?): lerna-semantic-release post > generate CHANGELOG"
+
+echo ">>> START: git add packages/**/CHANGELOG.md"
+git add packages/**/CHANGELOG.md
+echo ">>> DONE($?): git add packages/**/CHANGELOG.md"
+
+echo ">>> START: git commit -m \"chore(changelogs): update all CHANGELOG files\" -m \"[skip ci]\""
+git commit -m "chore(release): update all CHANGELOG files" -m "[skip ci]"
+echo ">>> DONE($?): git commit -m \"chore(changelogs): update all CHANGELOG files\" -m \"[skip ci]\""
+
+echo ">>> START: lerna-semantic-release perform > git push origin --tags and npm publish"
```

Comment:
sry for nitpick, but I personally prefer to have it in "npm scripts" command.

also it would be consistent with some of your other comments ( where it makes sense ofc )

consider:

```diff
-echo ">>> START: lerna-semantic-release perform > git push origin --tags and npm publish"`
+echo ">>> START: lerna-semantic-release perform > git push origin --tags && npm publish"
```